src/api/app/models/bs_request_action.rb

```diff
@@ -902,7 +902,7 @@ def source_access_check!
         # the target, the request creator (who must have permissions to read source)
         # wanted the target owner to review it
         tprj = Project.find_by_name(target_project)
-        if tprj.nil? || !User.current.can_modify_project?(tprj)
+        if tprj.nil? || User.current.nil? || !User.current.can_modify_project?(tprj)
```

Comment:
`BsRequestAction.source_access_check!` is also used in `BsRequestPermissionCheck`. Have you checked what this means?

Reply:
arrghhh